be/src/cloud/config.h

```diff
@@ -94,4 +94,6 @@ DECLARE_mBool(save_load_error_log_to_s3);
 // the theads which sync the datas which loaded in other clusters
 DECLARE_mInt32(sync_load_for_tablets_thread);
 
+DECLARE_mBool(txn_lazy_commit);
```

Comment:
Add `cloud_` prefix, and add a description too.

Reply:
> Add `cloud_` prefix, and add a description too.

done, add a cloud_ prefix